### ISSUE TITLE
Does not allow play

### ISSUE THREAD

**Issue by MXJosueDev** [idolize/xcloud-keyboard-mouse#126]
### Bug description

When you try to play I get a menu, when I click on the menu I get the menu again, making it impossible to play.

https://user-images.githubusercontent.com/69768362/224526228-fe497ffe-d080-4d4d-b1c3-826a57cb6d3c.mp4

### Extension Version

3.5

### What browser(s) are you seeing the problem on?

Chrome

### Browser version number

93.0.4577.107

**Comment by MXJosueDev** [comment]
Please, help

**Comment by broxenking** [comment]
what i would do is just restart the computer or refresh the browser your using

**Comment by idolize** [comment]
Yes, this is very odd. I've not seen this before. Have you tried restarting your computer and reinstalling the extension?
